.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -48,7 +48,11 @@ function clone_repo() {
     mkdir -p "$(dirname $LOCAL_PATH)"
 
     echo "BASE_BRANCH: $BASE_BRANCH"
-    git clone $GITHUB_PATH $LOCAL_PATH --branch $BASE_BRANCH
```

Comment:
rather than making this change - should this be cloning the `$UPSTREAM_BRANCH` instead? Since I think UPSTREAM_BRANCH is the base branch in the downstream (confusingly)

Reply:
Yeah, using `$UPSTREAM_BRANCH` should be working fine. I'll make the change

> confusingly - possibly we should rename the variable

we do have a mix in use of the terms "upstream" and "downstream" when referring those generated repos. We should standardize them at some point, but probably not in this PR.